h/static/scripts/permissions.coffee

```diff
@@ -37,10 +37,15 @@ module.exports = ['session', (session) ->
   # Typical use: annotation.permissions = permissions.private()
   ###
   private: ->
-    read: [session.state.userid]
-    update: [session.state.userid]
-    delete: [session.state.userid]
-    admin: [session.state.userid]
+    if not session?.state?.userid
```

Comment:
Also, looking at the tests, that's simply not true. A fake session object is already injected for all the `permissions` tests.